docs/sce_file_contents.md

```diff
@@ -1,21 +1,23 @@
 # Single-cell gene expression file contents
 
-Single-cell or single-nuclei gene expression data (unfiltered, filtered, or processed) is provided for use with R as an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
-This object contains the expression data, cell and gene metrics, associated metadata, and, in the case of multimodal data like ADTs from CITE-seq experiments, data from additional cell-based assays.
+Single-cell or single-nuclei gene expression data (unfiltered, filtered, or processed) is provided for use with R as an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) or with python as an HDF5 file containing an [`AnnData` object](https://anndata.readthedocs.io/en/latest/index.html).
```

Comment:
Would this be a little easier to consume with nested bullets? For example (apologies for almost-certainly-bad suggestion spacing...)

```suggestion
Single-cell or single-nuclei gene expression data (unfiltered, filtered, or processed) is provided in two formats:
  - As an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) for use in R
 - An HDF5 file containing an [`AnnData` object](https://anndata.readthedocs.io/en/latest/index.html) for use in Python
```